src/OpenSage.Game/Scripting/Actions/ActionLookup.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using ScriptAction = OpenSage.Data.Map.ScriptAction;
+using ScriptActionType = OpenSage.Data.Map.ScriptActionType;
+
+namespace OpenSage.Scripting.Actions
+{
+    public static class ActionLookup
+    {
+        // TODO?: This is technically unnecessary boilerplate; we could construct this table dynamically using reflection.
```

Comment:
Now that I think about it, since some games re-use the same identifier for a different action, we're going to have to build the table dynamically anyway.